de.fu_berlin.inf.dpp.stf/sonar-project.properties

```diff
@@ -1,14 +1,18 @@
 sonar.projectName        = Saros/STF
```

Comment:
This file is not needed anymore. See my comment in `de.fu_berlin.inf.dpp.stf.test/sonar-project.properties`